mkdocs.yml

```diff
@@ -82,19 +82,22 @@ theme:
   font:
     text: Inter
   palette:
+    # Palette toggle for automatic mode
+    - media: "(prefers-color-scheme)"
+      toggle:
+        icon: material/brightness-auto
+
+    # Palette toggle for light mode
     - media: "(prefers-color-scheme: light)"
       scheme: default
-      accent: cyan
       toggle:
-        icon: kolena/light-mode-20
-        name: "Switch to dark mode"
```

Comment:
I have removed these `name`s here. mkdocs appears to have a bug where the wrong label is being rendered for this toggle when there are three items